### ISSUE TITLE
Expand KIN to include exactMatches for all HL7 FamilyMemberRelationship Codes

### ISSUE THREAD

**Issue by tripb** [GA4GH-Pedigree-Standard/family_history_terminology#50]
Per discussion at Clin/Pheno Data Capture Workstream meeting, we would like to expand KIN to also include the more specific and more general relationships contained within the HL7 RoleCode value set so that KIN is a superset of HL7 RoleCode, containining at minimum 1:1 matches for all RoleCodes.

Will expand so that all narrowMatch mappings also have a related exactMatch (also keeping the narrowMatch).

Questions:
@cmungall or others - Do we want to try to keep the more 'clinical' wording typically found in KIN or adopt some of the HL7 more colloquial wordings for these?  
       For example, we currently have narrowMatch for KIN:013 `isParentalSiblingOf` --> `AUNT`.  Should the exactMatch be `isParentalFemaleSiblingOf` or `isAuntOf` ?     
       It gets a bit more complex for `BROINLAW` (brother-in-law).  KIN has the broader `isSocialLegalRelativeOf`.  Should this be something like `isSocialLegalBrotherInLawOf`, `isSocialLegalMaleSiblingInLawOf` or we just go with `isBrotherInLawOf`?
      Do we have conventions we are trying to align with here?

Does KIN have a hierarchy specified somewhere?  eg to show that `isSiblingOf` encompasses `isMaleSiblingOf` ?  Should it?
HL7 RoleCodes does this using the concept of 'levels' and through consistent naming of the codes.

**Comment by cmungall** [comment]
> For example, we currently have narrowMatch for KIN:013 isParentalSiblingOf --> AUNT

I think we can remove this. We should only retain inexact mappings for cases where more precise mappings cannot be found

> isParentalFemaleSiblingOf or isAuntOf

Hmm, in the current release version I see

KIN:060 ! isMaternalAuntOf
KIN:061 ! isPaternalAuntOf

but I don't see a grouping for these. I think we should create a grouping isAuntOf and have that be an exact match to AUNT

**Comment by cmungall** [comment]
From @lindsmith:

> I think one of the main reasons we didn’t use colloquial wording like “aunt” is because it doesn’t well accommodate our global audience

Good point. 

>  For that reason, I’m inclined to suggest we go with something like is “ParentalFemaleSiblingOf”

Yes, that sounds like a good primary label for the concept.

Can we still include "aunt of" as a synonyms? We can also categorize synonyms, e.g. according to culture

**Comment by monicacecilia** [comment]
> Can we still include "aunt of" as a synonyms? We can also categorize synonyms, e.g. according to culture

Yes, that's a great idea. Let's add that. Can we do a review of the existing elements and collect synonyms? We can start with a first pass amongst our teams and then circulate to the broader GA4GH community ahead of next month's Pedigree/FHH convo on the Clin/Pheno call (12/18). 

Could someone please start that?
@cmungall @tripb 👀 👆

**Comment by tripb** [comment]
@monicacecilia , @cmungall - I'm happy to get this started.  My plan is to create a `kin_concept_relationship.csv` which will contain hierarchy (eg “ParentalFemaleSiblingOf” to "ParentalSiblingOf") and synonyms by language or culture (eg "Aunt" and "Tia" to “ParentalFemaleSiblingOf”).  I'll have a column for `predicate_id` = `[skos:broader|skos:narrower|skos:altLabel]` and a column for `ex:context` = `[english_colloquial|spanish_colloquial|etc]` and `subject_id`, `subject_label`, `object_id`, `object_label` and `object_label` We can tweak how we structure this (prob using SKOS) but at least I'll get the content started.

**Comment by cmungall** [comment]
perfect, thanks!

**Comment by tripb** [comment]
Proposing several additions to KIN and updates to HL7 mappings, as well as starting colloquial labels: https://github.com/ga4gh/pedigree_family_history_terminology/pull/51
